advisor/Vector_Tutorial_Data_Alignment/cpp/cpp.json

```diff
@@ -0,0 +1,16 @@
+{
+  "projectOptions": [ 
+    {
+      "projectType": "makefile",
+      "dockerSupported": "false"
+    } 
+  ],
+  "makeVariables": {
+    ".DEFAULT_GOAL": "default",
+    "--directory":"DataAlignment"
+  },
+  "indexerVariables": {
+    "COMMAND": "${ISS_ROOT}/compilers_and_libraries/linux/bin/intel64/icpc",
+    "FLAGS": "-I ${ISS_ROOT}/advisor_2019/include/"
```

Comment:
I had been handling include paths by setting the CPATH environment variable, partially so that the value is available if we enable non-Makefile build types in the future, and partially because I added indexerVariables late. Do you think it makes a difference?

Reply:
I'm not sure, I'll have to check the code and/or test.